index.html

```
@@ -530,6 +530,56 @@ <h5 class="slide__heading">
+                            ></span>
+                            <input
+                                type="email"
+                                name="email"
+                                id="email"
+                                class="newsletter__email"
+                                placeholder="Your Email"
+                            />
+                        </div>
+                        <input
+                            class="btn btn--primary btn--expanded"
+                            class="btn--subscribe"
```

HTMLHint (1.5.0)

[error] Duplicate of attribute name [ class ] was found.

(attr-no-duplication)